content/_changelogs/11.0.0.md

```diff
@@ -4,6 +4,9 @@ _Released MM/DD/YYYY_
 
 **Breaking Changes:**
 
+- `experimentalSessionAndOrigin` flag has been removed and all functionality is
+  on by default now.
+- `testIsolation` defaults to `strict` now.
```

Comment:
@emilyrohrbough is this `strict` or `on`? I guess this changes pending https://github.com/cypress-io/cypress/pull/24316?
```suggestion
- `testIsolation` defaults to `on` now.
```

Reply:
@AtofStryker This will be `on` once my changes merge. I'll have to update once that PR is merged